light-cli/src/commands/config/index.ts

```diff
@@ -0,0 +1,86 @@
+import { Command, Flags } from "@oclif/core";
+import * as fs from "fs";
+import { CustomLoader } from "../../utils/utils";
+
+class ConfigCommand extends Command {
+  static description = "Update the configuration values";
+
+  static flags = {
+    rpcUrl: Flags.string({
+      char: "r",
+      description: "Solana RPC URL",
+    }),
+    relayerUrl: Flags.string({
+      char: "l",
+      description: "Relayer URL",
+    }),
+    secretKey: Flags.string({
+      char: "s",
+      description: "Secret key in string format",
+    }),
+    relayerRecipient: Flags.string({
+      char: "u",
+      description: "Relayer recipient",
+    }),
+    lookupTable: Flags.string({
+      char: "t",
+      description: "Look-up table",
+    }),
+    payer: Flags.string({
+      char: "p",
+      description: "Payer secret key",
+    }),
+  };
+
+  protected finally(_: Error | undefined): Promise<any> {
+    process.exit();
+  }
+
+  static examples = [
+    "$ light config --rpcUrl https://solana-api.example.com",
+    "$ light config --relayerUrl https://relayer.example.com",
+    "$ light config --secretKey your-secret-key-in-base58-string-format",
```

Comment:
pls add that this is a solana keypair secretkey

Reply:
Done!